### ISSUE TITLE
The image for the IRV lab can not download from Google Drive

### ISSUE THREAD

**Issue by JerrySOET** [LoCO-AUV/loco_config#1]

**Comment by campey** [comment]
Yes, there was a permissions issue when I tried to. I mailed the team and they provided this updated link:
https://drive.google.com/drive/folders/1Itb1DBPN1fRqZbki9KhZ-fKaBIshAbB9?usp=sharing

Perhaps we can update the wiki with this link?
